stdlib/ballerina-config/src/main/java/org/ballerinalang/config/cipher/AESCipherTool.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.config.cipher;
+
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ * This tools is used to encrypt and decrypt data using AES Algorithm CBC mode and PKCS #5 padding.
+ *
+ * @since 0.964.0
+ */
+public class AESCipherTool {
+
+    private static final String ALGORITHM_AES_CBC_PKCS5 = "AES/CBC/PKCS5Padding";
+    private static final String ALGORITHM_AES = "AES";
+    private static final String ALGORITHM_SHA_256 = "SHA-256";
+    private static final String DEFAULT_INITIALIZATION_VECTOR = "0123456789ABCDEF";
```

Comment:
If we incorporate the random IV related suggestion below, we can get rid of this value.

Reply:
Yes. Will make the necessary changes